apps/tsilhqotin-language-hub/src/Components/Pages/Contact/validation/validateContactInfo.ts

```diff
@@ -0,0 +1,31 @@
+import ContactInfo from '../contactData/ContactInfo';
+
+// TODO Import this from the validation lib
+const isStringWithNonzeroLength = (input: unknown): input is string =>
+    typeof input === 'string' && input.length > 0;
+
+export const isErrorArray = (input: unknown): input is Error[] =>
+    Array.isArray(input) && input.every((item) => item instanceof Error);
+
+export default (input: unknown): ContactInfo | Error[] => {
+    const { name, title, department } = input as ContactInfo;
+
+    const allErrors: Error[] = [];
+
+    if (!isStringWithNonzeroLength(name))
+        allErrors.push(new Error(`Contact info: name must be a string. Received: ${name}`));
```

Comment:
We really need to be able to import `@coscrad/validation` for this. But I'll be solving that problem on my current project, so wait for me so not to duplicate efforts.